Velocity_Model/scripts/get_z.py

```diff
@@ -31,6 +31,6 @@
 
     stat_df = formats.load_station_file(args.ll)
 
-    output_file = args.output / "z.csv"
+    output_file = args.output
```

Comment:
The help asks for a filepath rather than a directory, so this change puts that in-line with the help text.